lib/ts/recipe/emailverification/components/features/emailVerification/index.tsx

```diff
@@ -66,7 +66,7 @@ export const EmailVerification: React.FC<Prop> = (props) => {
         const token = getQueryParams("token") ?? undefined;
         if (token === undefined) {
             if (!sessionContext.doesSessionExist) {
-                await SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history);
+                await SuperTokens.getInstanceOrThrow().redirectToAuth({ history: props.history });
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.